docs/source/model_design/vivarium_features/cause_of_death_selection/index.rst

```diff
@@ -0,0 +1,216 @@
+With cause of death, we don't actually run a scenario in which we delete that cause;
+the cause of death is assigned within the scenario.
+
+To be internally consistent, though, we **want the deaths we assign cause A on a given timestep
+to be the same deaths that would not occur if we did run a no-cause-A scenario.**
+This might seem like an academic point, since we will never run such a scenario.
+But this property turns out to be equivalent to another property that has more practical relevance:
+**when we intervene only on cause A, only cause A deaths should be averted**.
+
+Current behavior
+--------------------------
+
+As of March 2025, the internal consistency described above is not achieved.
+It is possible to have two scenarios in which only cause A is intervened on,
+but cause B deaths are averted when comparing the two scenarios.
+
+.. note::
+
+  This is **only** due to random noise (stochastic uncertainty) about the number
+  of cause B deaths averted.
+  *In expectation* -- that is, as the simulated population size goes to infinity --
+  the number of cause B deaths averted is 0.
+  In other words, this is an issue with our common random numbers, which are intended
+  to reduce the variance in our estimates of averted quantities.
+
+To briefly describe the reason why this can occur:
+in Vivarium Public Health's mortality component,
+two random decisions are made: first, we decide whether or not the simulant dies,
+and only then do we decide the cause of death, with two separate random draws.
+
+This is conceptually equivalent to (but possibly computationally faster than)
+making a single choice between:
+
+1. The simulant dies of cause A
+2. The simulant dies of cause B
+3. ... (and so on for other causes in the model)
+4. The simulant dies of an unmodeled cause
+5. The simulant does not die
+
+Where each option is assigned a probability, and the probabilities collectively sum to 1.
+
+We can visualize this as follows. For each simulant (with the same age and sex,
+who has causes A, B, and C), Vivarium chooses a random point on the following
+stacked bar chart:
+
+.. image:: cod_diagram.drawio.svg
+
+The problem is that when one of the cause-specific probabilities changes between scenarios,
+it *shifts* all the rest.
+Here's an example where an intervention reduces the excess mortality rate of cause A:
+
+.. image:: cod_issue_diagram.drawio.svg
+
+Instead of some simulants who would have died of cause A not dying at all,
+some simulants who would have died of cause A die of cause B instead,
+some who would have died of cause B die of cause C instead,
+and some who would have died of cause C don't die at all.
+Cause B and C don't change size -- so in expectation, the number of deaths is the same --
+but depending on where our finite population of simulants actually fall on this
+graph, we may get nonzero (positive or negative) numbers of deaths averted
+for both cause B and C.
```

Comment:
Can we add a bit more about when this is more of an issue and when we're most likely to see it? E.g., having multiple chronic conditions and having lots of simulants who have high comorbidity rates.